requirements.txt

```diff
@@ -1,9 +1,9 @@
-pyqt5==5.11.3
+pyqt5==5.10
```

Comment:
Why downgrade version?

Reply:
It is better to keep some fixed version on all the dependencies, since it minimizes problems when one or more packages update on their own. Update project dependencies when they are required.

Reply:
need pyqt5==5.10 for osx. otherwise cannot build.